flag_int64_slice.go

```diff
@@ -33,6 +33,18 @@ func (i *Int64Slice) Set(value string) error {
 		return nil
 	}
 
+	if strings.Contains(value, ",") {
```

Comment:
Nit: I think the ergonomics around `strings.Contains` means we can skip the conditional and always range.

- `strings.Split("1", ",")` -> `[1]`
- `strings.Split("1,2", ",")` -> `[1 2]`

Meaning we can drop the `if` statement, outdent everything in there, and remove the logic for treating `value` as a non-list value.

Reply:
Fixed this